inc/template-tags.php

```diff
@@ -114,7 +114,7 @@ function twentyseventeen_edit_link( $id ) {
 	} elseif ( get_post( $id ) ) {
 		$type = __( 'Post', 'twentyseventeen' );
 	}
-	$link = edit_post_link(
```

Comment:
@codersantosh Just a tip, avoid putting changes in the **master** branch of your fork. Create a branch off of master for each separate pull request. This will help break up edits and make for easier merges. :)